shared/src/types/time.rs

```diff
@@ -98,6 +102,18 @@ impl DateTimeUtc {
     pub fn now() -> Self {
         Self(Utc::now())
     }
+
+    /// Returns an rfc3339 string or an error.
+    pub fn to_rfc3339(&self) -> Result<String, std::io::Error> {
+        Time::try_from(*self)
```

Comment:
Looks like we could use `chrono::DateTime::to_rfc3339` directly without having to go through Tendermint? Something like
```rust
return DateTime::to_rfc3339(&self.0)
```
then we can avoid having to return an error from this function as well

Reply:
:+1: